mora/validator.py

```diff
@@ -104,11 +106,16 @@ def _get_active_validity(reg: dict) -> typing.Mapping[str, str]:
 
 def is_date_range_in_org_unit_range(org_unit_uuid, valid_from, valid_to):
     scope = lora.Connector(
-        virkningfra=util.to_lora_time(valid_from),
-        virkningtil=util.to_lora_time(valid_to)
+        virkningfra=util.to_lora_time(util.NEGATIVE_INFINITY),
+        virkningtil=util.to_lora_time(util.POSITIVE_INFINITY)
```

Comment:
Why was this change made?

Reply:
The trick is that the termination check can query a unit _prior_ to its creation — hence, merely obtaining the validity at the date of the check yields little useful data, and prevents us from returning the helpful _actual_ validity.

Reply:
Ah, so this is purely to be able to return more helpful error messages?

In any case, I would like this function to be as similar to `is_date_range_in_employee_range` as possible, so it would be nice if similar adjustments could be made there as well - also in terms of the added not-found check